APMrover2/crash_check.cpp

```diff
@@ -21,15 +22,32 @@ void Rover::crash_check()
   // TODO : Check if min vel can be calculated
   // min_vel = ( CRASH_CHECK_THROTTLE_MIN * g.speed_cruise) / g.throttle_cruise;
 
-  if ((ahrs.groundspeed() >= CRASH_CHECK_VEL_MIN) ||        // Check velocity
+ if ((ahrs.groundspeed() >= CRASH_CHECK_VEL_MIN) ||        // Check velocity
       (fabsf(ahrs.get_gyro().z) >= CRASH_CHECK_VEL_MIN) ||  // Check turn speed
       (fabsf(g2.motors.get_throttle()) < CRASH_CHECK_THROTTLE_MIN)) {
-    crash_counter = 0;
-    return;
-  }
+      crash_counter = 0;
+      bool crash_checker = true;
+
+
+  float lean_angle = degrees(safe_sqrt(ahrs.pitch*ahrs.pitch + ahrs.roll*ahrs.roll));
+  if (lean_angle > CRASH_ANGLE) {
+    crash_counter ++;
```

Comment:
The indentation in the coding style is 4 white Spaces. There are other lines as well.